scripts/version_mapper.py

```diff
@@ -17,22 +19,26 @@ def sort_versions_descending(versions_list):
     return versions_list[::-1]
 
 
-def update_switch_version_file(filename, new_version, cname, render_last):
+def update_switch_version_file(
+    json_filename, new_version, cname, render_last, annoucement_filename
```

Comment:
```suggestion
    json_filename, new_version, cname, render_last, announcement_filename
```

Reply:
Ops! Good catch, @Revathyvenugopal162. I'm likely to have introduced this through the usage of my LSP. Let me fix it.